src/openvpn/errlevel.h

```diff
@@ -148,6 +148,8 @@
 #define D_PF_DEBUG           LOGLEV(7, 72, M_DEBUG)  /* packet filter debugging, must also define PF_DEBUG in pf.h */
 #define D_PUSH_DEBUG         LOGLEV(7, 73, M_DEBUG)  /* show push/pull debugging info */
 
+#define D_VLAN_DEBUG         LOGLEV(7, 72, M_DEBUG)  /* show VLAN tagging/untagging debug info */
```

Comment:
I'll make sure it's sorted, but I don't know whether 72 makes sense in this context.  I think it was originally chosen to be unique, but since then other things have been added.  I assume the intention was to allow these messages to be muted without muting anything else.  Does it matter that they're at the same mute-level as D_PF_DEBUG?  If so, what would be an appropriate value (and would we use a different value if we add additional logging as you mentioned in another comment)?

Reply:
Please see Gert's comment below